Panic - invalid memory address

### Background
Built from source in master branch
Getting the following stack trace when started lnd daemon and after unlocking wallet:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x804a8ac]

goroutine 1 [running]:
runtime/internal/atomic.Xadd64(0xb4b8944, 0x1, 0x0, 0xa60cb3f0, 0x5c)
	/usr/local/go/src/runtime/internal/atomic/asm_386.s:105 +0xc
github.com/btcsuite/btcwallet/chain.(*BitcoindConn).NewBitcoindClient(0xb4b8900, 0xb7124e0)
	/home/go/pkg/mod/github.com/btcsuite/btcwallet@v0.11.1-0.20210405201449-683061f79797/chain/bitcoind_conn.go:501 +0x58
github.com/lightningnetwork/lnd/chainntnfs/bitcoindnotify.New(0xb4b8900, 0x97781c0, 0x904f1a0, 0xb70a720, 0x904f180, 0xb70a720, 0xe)
	/home/go/src/github.com/lightningnetwork/lnd/chainntnfs/bitcoindnotify/bitcoind.go:92 +0x106
github.com/lightningnetwork/lnd/chainreg.NewChainControl(0xbcd9ef0, 0xb70a710, 0xbca6501, 0xb896500)
	/home/go/src/github.com/lightningnetwork/lnd/chainreg/chainregistry.go:411 +0x19a1
github.com/lightningnetwork/lnd.Main(0xb89aa00, 0x0, 0x0, 0x0, 0x0, 0xb4abc40, 0xb43da40, 0xb43da80, 0xb43dac0, 0x0, ...)
	/home/go/src/github.com/lightningnetwork/lnd/lnd.go:551 +0xf25
main.main()
	/home/go/src/github.com/lightningnetwork/lnd/cmd/lnd/main.go:36 +0x1ac
```

### Your environment

```
{
    "lncli": {
        "commit": "v0.12.0-beta-340-g213b264e",
        "commit_hash": "213b264e4c0f92a23c7dd8a42e0b68e4958a7e3a",
        "version": "0.12.99-beta",
        "app_major": 0,
        "app_minor": 12,
        "app_patch": 99,
        "app_pre_release": "beta",
        "build_tags": [
        ],
        "go_version": "go1.15.6"
    },
    "lnd": null
}
```
4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:00 UTC 2019 i686 i686 i686 GNU/Linux
18.04

I tired to pull and rebuild btcsuites/btcwallet client:
```
cd ~/gocode/src/github.com/btcsuite/btcwallet/
git pull
git log # shows Merge pull request #742 from wpaulino/bitcoind-conn-atomic-alignment chain: fix alignment of atomic integers in bitcoind conn struct
make clean && make && make install
```
then rebuild LND:
```
(cd $GOPATH/src/github.com/lightningnetwork/lnd && git status && git pull && make clean && make && make in
stall)
```

and still getting the error:
```
2021-04-16 10:05:59.281 [INF] LNWL: Started listening for bitcoind transaction notifications via ZMQ on 127.0.0.1:29001
2021-04-16 10:05:59.311 [INF] LTND: Shutdown complete

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4 pc=0x1326c]

goroutine 1 [running]:
github.com/btcsuite/btcwallet/chain.(*BitcoindConn).NewBitcoindClient(0x2582100, 0x2678ae0)
        /home/lightning/gocode/pkg/mod/github.com/btcsuite/btcwallet@v0.11.1-0.20210405201449-683061f79797/chain/bitcoind_conn.go:501 +0x50
github.com/lightningnetwork/lnd/chainntnfs/bitcoindnotify.New(0x2582100, 0x19aaaa0, 0x10e0cd0, 0x263c000, 0x10e0cb0, 0x263c000, 0xe)
        /home/lightning/gocode/src/github.com/lightningnetwork/lnd/chainntnfs/bitcoindnotify/bitcoind.go:92 +0x108
github.com/lightningnetwork/lnd/chainreg.NewChainControl(0x2627ed0, 0x2499900, 0x26c4501, 0x26d6500)
        /home/lightning/gocode/src/github.com/lightningnetwork/lnd/chainreg/chainregistry.go:411 +0x1834
github.com/lightningnetwork/lnd.Main(0x2606500, 0x0, 0x0, 0x0, 0x0, 0x25e7900, 0x24402c0, 0x2440300, 0x2440340, 0x0, ...)
        /home/lightning/gocode/src/github.com/lightningnetwork/lnd/lnd.go:551 +0xcac
main.main()
        /home/lightning/gocode/src/github.com/lightningnetwork/lnd/cmd/lnd/main.go:36 +0x1ac
```

I'm 75% sure the following does not matter, yet in earlier attempt to fix the problem I upgraded from 32-bit OS `bitcoind 0.19.1` to 64-bit OS and 64-bit build of `bitcoind 0.20.1`

--- a/go.mod
+++ b/go.mod
@@ -9,7 +9,7 @@ require (
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/btcsuite/btcutil/psbt v1.0.3-0.20201208143702-a53e38424cce
-	github.com/btcsuite/btcwallet v0.11.1-0.20210405201449-683061f79797
+	github.com/btcsuite/btcwallet v0.11.1-0.20210413223001-fe61cc5d7886
 	github.com/btcsuite/btcwallet/wallet/txauthor v1.0.1-0.20210329233242-e0607006dce6
 	github.com/btcsuite/btcwallet/wallet/txrules v1.0.0
 	github.com/btcsuite/btcwallet/wallet/txsizes v1.0.1-0.20210329233242-e0607006dce6 // indirect

